### PR TITLE
Settings ivanka

### PR DESCRIPTION
**Pull request by hamsun7**

### CODE REVIEW

File: src/app/kingdom-settings/kingdom-settings.service.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export class KingdomSettingsService {
   constructor(private kingdomService: KingdomService) { }
 
   getSettings(): Observable<any> {
+    console.log(this.kingdomService.getKingdom());
```
Comment (gyopak) [inline src/app/kingdom-settings/kingdom-settings.service.ts:20]: `log`

File: src/app/kingdom-settings/kingdom-settings.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should call the getSettings function', () => {
+ //    let kingdom: Kingdom;
+     spyOn(kingdomService, 'getKingdom');
+     service.getSettings();
+     expect(kingdomService.getKingdom).toHaveBeenCalled();
+//    service.getSettings().subscribe(response => {
```
Comment (gyopak) [inline src/app/kingdom-settings/kingdom-settings.service.spec.ts:41]: comments

File: src/app/kingdom-settings/kingdom-settings.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should call the getSettings function', () => {
+ //    let kingdom: Kingdom;
+     spyOn(kingdomService, 'getKingdom');
+     service.getSettings();
+     expect(kingdomService.getKingdom).toHaveBeenCalled();
```
Comment (adamgyulavari) [inline src/app/kingdom-settings/kingdom-settings.service.spec.ts:40]: @hamsun7 this test doesnt test anything

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by adamgyulavari** [comment]
@hamsun7 resolve conflicts

**Pull request closed without merging**